internal/server/v0/placestatvar/golden/get_place_stat_vars/alb.json

```diff
@@ -7263,7 +7263,6 @@
         "sdg/AG_FOOD_WST_PC.FOOD_WASTE_SECTOR--FWS_OOHC",
         "sdg/AG_FOOD_WST_PC.FOOD_WASTE_SECTOR--FWS_RTL",
         "sdg/AG_FPA_CFPI",
-        "sdg/AG_LND_DGRD",
```

Comment:
Are these deletions expected @ajaits

Reply:
The recent UN SDG data update removed estimated data sets, so this seems expected.